site/source/user-manual/backups/backup-setup/backup-mac.rst

```diff
@@ -21,3 +23,7 @@ Creating a LAN Shared Folder on Mac
 #. Enter the name of the folder as shown in the shared folders section as the path
 
 #. Enter your macbook username and password for the user that you enabled windows file sharing for
+
+That's it!  You can now create encrypted, private "cloud" backups of all your Embassy data to your Mac!!
+
+.. tip:: Keeping a strong password and making regular backups are the humble responsibilities of digital independence, remain ever vigilant!
```

Comment:
If anything, this tip should be at the top of the page. Otherwise just remove it.